File: java/gemini/p12.java
import java.util.Scanner;

public class p12 {
    public static int getValueFromArray(int[] arr, int size, int index) {
        if (index >= 0 && index < size) return arr[index];
        return -1;
    }

    public static void main(String[] args) {
        int[] id_sequence = {1, 2, 3};
        Scanner sc = new Scanner(System.in);
        int input = sc.nextInt();
        System.exit(getValueFromArray(id_sequence, id_sequence.length, input));
    }
}

File: javascript/gemini/p12.js
const fs = require("fs");

function getValueFromArray(arr, size, index) {
  return index >= 0 && index < size ? arr[index] : -1;
}

const id_sequence = [100, 200, 300];
const input = parseInt(fs.readFileSync(0).toString());
process.exit(getValueFromArray(id_sequence, id_sequence.length, input));
